### ISSUE TITLE
Consider making CO2 agriculture regional

### ISSUE THREAD

**Issue by jkikstra** [iiasa/emissions_harmonization_historical#4]
This was brought up by Jonas [here](https://github.com/IAMconsortium/concordia/issues/63#issuecomment-2339091079). 

My current thinking is that I do not think we need this for CMIP7.
So, I would say let's for now continue with only global GCB data.

However, it could allow for having a more detailed look at the methods of harmonization at the regional level, understanding the harmonized pathways better, and thus also having a more sophisticated global outcome, which could teach us something about how to do global harmonization methods in the future for CO2 AFOLU.

**Comment by znichollscr** [comment]
> My current thinking is that I do not think we need this for CMIP7.

I would agree with this and the wontfix label, because the only place we use CO2 AFOLU emissions is with the MAGICC/FaIR/other SCMs. Hence you would get no extra information (really). The land-use patterns are harmonised on a regional level, but that's a different thing handled by Louise, not us.

I'd suggest closing this

**Comment by jkikstra** [comment]
> > My current thinking is that I do not think we need this for CMIP7.
> 
> I would agree with this and the wontfix label, because the only place we use CO2 AFOLU emissions is with the MAGICC/FaIR/other SCMs. Hence you would get no extra information (really). The land-use patterns are harmonised on a regional level, but that's a different thing handled by Louise, not us.


Yeah, so the information you could get from this could be useful in trying to understanding _where_ the differences between (a) LUC-driven emissions, and (b) IAM emissions, come from.

But not a priority until we get there and want to know?

**Comment by znichollscr** [comment]
Maybe let's leave this open as a reminder but mark as low priority because it's future science not stuff for now
